hw/BUILD

```diff
@@ -4,6 +4,19 @@
 
 load("//rules:fusesoc.bzl", "fusesoc_build")
 
+# This configuration causes fusesoc to produce a single-threaded Verilator model
```

Comment:
Does it configure it to produce a single threaded model? I'm having trouble knowing which thing overloads what, I see how you're setting it to run on 1 thread and that looks good, and I assume the default is to run on 4, but I'm getting confused when the first thing to happen below sets it to run in 4 threads. I can't tell if I'm misreading it or the comment just needs to be updated